src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -212,14 +212,14 @@ private ImmutableArray<TypeSymbolWithAnnotations> GetDeclaredConstraintTypes()
                             continue;
                         }
 
+                        var type = TypeSymbolWithAnnotations.Create(typeSymbol, isNullableIfReferenceType: null);
```

Comment:
>null [](start = 107, length = 4)

Why null-oblivious?

Reply:
`ModuleSymbol.UtilizesNullableReferenceTypes` indicates whether the module was compiled with the feature enabled. If the feature was not enabled, reference types should be null-oblivious.

---
In reply to: [195545642](https://github.com/dotnet/roslyn/pull/27525#discussion_r195545642) [](ancestors = 195545642)